docs/2021-ga-release-blog-post.md

```diff
@@ -0,0 +1,58 @@
+We're excited to announce the new GA release for Javascript SDKs. These SDKs have several new features that help our customers to have a great experience in using the Azure Services. Some of the important SDKs that have been released are:
+
+1. [Compute](https://www.npmjs.com/package/@azure/arm-compute/v/17.0.0)
+2. [Network](https://www.npmjs.com/package/@azure/arm-network/v/26.0.0)
+3. [Storage](https://www.npmjs.com/package/@azure/arm-storage/v/17.0.0)
+4. [Sql](https://www.npmjs.com/package/@azure/arm-sql/v/9.0.0)
+5. [Resources](https://www.npmjs.com/package/@azure/arm-resources/v/5.0.0)
+6. [Policy](https://www.npmjs.com/package/@azure/arm-policy/v/5.0.0)
+7. [Managed applications](https://www.npmjs.com/package/@azure/arm-managedapplications/v/2.0.0)
+8. [Links](https://www.npmjs.com/package/@azure/arm-links/v/2.0.0)
+9. [Locks](https://www.npmjs.com/package/@azure/arm-locks/v/2.0.0)
+10. [Features](https://www.npmjs.com/package/@azure/arm-features/v/3.0.0)
+
+Each of these SDKs follows the new Azure SDK guidelines. All these SDKs have been generated using the latest version of the new [Autorest Typescript SDK Generator](https://www.npmjs.com/package/@autorest/typescript/v/6.0.0-beta.15). These libraries provide new authentication mechanisms, uses Promises, etc. We encourage you to try out the libraries and provide your feedback.
+
+## Key concepts
+
+The new libraries have the following changes:
+
+1. **Authentication**: The packages `@azure/ms-rest-nodeauth` or `@azure/ms-rest-browserauth` are replaced with a new package `@azure/identity` instead.
+2. **Promises**: Method overloads that use callbacks have been replaced to use Promise instead.
+3. The results of List operations could be iterated by using the `PagedAsyncIterableIterator` interface
+4. The SDKs have been changed to support ECMAScript 2015 (ES6) and beyond
+
+A detailed list and usage of these features could be found in the [migration guide](https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/MIGRATION-guide-for-next-generation-management-libraries.md)
```

Comment:
@qiaozha , do we have the aka.ms link for the migration guide? it is better to use that in case we update the file name/location in the future.

Reply:
yeah. @sarangan12  You may use the aka.ms links below in your post.
 Changelog: https://aka.ms/js-track2-changelog

Migration Guide: https://aka.ms/js-track2-migration-guide

Quick Start: https://aka.ms/js-track2-quickstart

Reply:
I have added the aka.ms link for the migration guide